packages/marketplace/src/components/pages/admin-apps/admin-apps.tsx

```diff
@@ -125,15 +127,11 @@ export const generateColumns = ({ dispatch, setDataDeleteModal, deleteModalData
   ]
 }
 
-export const refreshForm = (onSubmit: Function, resetForm: Function) => () => {
-  resetForm()
-  onSubmit({ appName: '', companyName: '', developerName: '' })
+export const refreshForm = history => () => {
+  history.push('apps')
```

Comment:
Please avoid hardcoded strings, `apps` should be moved to `routes.ts`

Reply:
I changed it. Use Routes config now